bsdploy/bootstrap_utils.py

```diff
@@ -281,16 +291,17 @@ def install_pkg(self, root, chroot=None, packages=[]):
         if pkg is not None:
             print("\nInstalling pkg")
             if not exists(pkg.local):
-                run('fetch -o {0.remote} {0.url}'.format(pkg))
+                run(self.env_vars + 'fetch -o {0.remote} {0.url}'.format(pkg), shell=False)
```

Comment:
yes, exactly. basically. setting shell to false was the only reliable way for me to set the environment variable for fetch etc. to pick it up.

fabric's built in context manager assumes bash and doesn't work here